stdlib/multiprocessing/__init__.pyi

```diff
@@ -29,6 +29,33 @@ if sys.version_info >= (3, 8):
 if sys.platform != "win32":
     from multiprocessing.context import ForkContext, ForkServerContext
 
+# fmt: off
```

Comment:
I'd rather we just let Black format this.

Reply:
Okay!

Reply:
Thanks! (pet peeve: it's "free rein", as in letting go of your horse's reins)

Reply:
Lol, of course. Too much writing of code recently, not enough writing of words...